fossasia-histogram/index.html

```diff
@@ -22,65 +22,70 @@
     <script src="/js/angular.min.js"></script>
     <style>
       .search-tweets {
-      	width:800px;
-      	display: -webkit-flex;
-	    display: flex;
-	    -webkit-align-items: stretch;
-	    align-items: stretch;
-	    margin-left:350px;	
-	  }
-	   .search-tweets input{
-		  width:600px;
-		  height: 45px;
-		  padding:8px;
-	  }
-	  .search-btn{
-	    background-color: #d81b60;
-	    color: white;
-	    width: 50px;
-	    height: 45px;
-	  }
-	  .loader {
-		  border: 8px solid #f3f3f3;
-		  border-radius: 50%;
-		  border-top: 8px solid #D81B60;
-		  width: 100px;
-		  height: 100px;
-		  -webkit-animation: spin 2s linear infinite;
-		  animation: spin 2s linear infinite;
-		  position: relative;
-		  top:180px;
-		  left:-320px;
-	  }
+        width:800px;
+        display: -webkit-flex;
+        display: flex;
+        -webkit-align-items: stretch;
+        align-items: stretch;
+        margin-left:350px;  
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/fossasia-histogram/index.html
+++ b/fossasia-histogram/index.html
@@ -27,7 +27,7 @@
         display: flex;
         -webkit-align-items: stretch;
         align-items: stretch;
-        margin-left:350px;  
+        margin-left:350px;
       }
        .search-tweets input{
           width:600px;
```